show/main.py

```diff
@@ -2081,6 +2081,98 @@ def ztp(status, verbose):
        cmd += ["--verbose"]
     run_command(cmd, display_cmd=verbose)
 
+#
+# 'bmp' group ("show bmp ...")
+#
+@cli.group(cls=clicommon.AliasedGroup)
+def bmp():
+    """Show details of the bmp dataset"""
+    pass
+
+# 'bgp-neighbor-table' subcommand ("show bmp bgp-neighbor-table")
+@bmp.command('bgp-neighbor-table')
+@clicommon.pass_db
+def bmp_neighbor_table(db):
+    """Show bmp bgp-neighbor-table information"""
+    bmp_headers = ["Neighbor_Address", "Peer_Address", "Peer_ASN", "Peer_RD", "Peer_Port", "Local_Address", "Local_ASN", "Local_Port",
+                   "Advertised_Capabilities", "Received_Capabilities"]
+
+    bmp_keys = db.db.keys(db.db.STATE_DB, "BGP_NEIGHBOR_TABLE|*")
+
+    click.echo("Total number of bmp neighbors: {}".format(0 if bmp_keys is None else len(bmp_keys)))
+
+    bmp_body = []
+    if bmp_keys is not None:
+        for key in bmp_keys:
+            key_values = key.split('|')
+            values = db.db.get_all(db.db.STATE_DB, key)
+            bmp_body.append([values["peer_addr"], values["peer_addr"], values["peer_asn"], values["peer_rd"], values["peer_port"],
```

Comment:
>peer_addr [](http://example.com/codeflow?start=37&length=9)

It's very easy to make one order mistake, and it is also difficult to expose the bug in test. Is there a systematically one to reduce this kind of mistake by refactoring?